src/content/about-the-data/data-validation-of-source-systems.mdx

```diff
@@ -1,8 +1,14 @@
-USAspending.gov is operated by the Treasury Department and receives data
-from a variety of government source systems outside of Treasury. USAspending.gov
-does not separately validate the data collected on external source systems;
-USAspending.gov relies on the data validation and internal controls of
-these external source systems. More information about Broker validation
-rules and SAO certification can be found in the "DAIMS Validation Rules"
-and "DAIMS Practices and Procedures" links on the
-<a target="_blank" rel="noopener noreferrer" href="https://fiscal.treasury.gov/data-transparency/resources.html">resources page for the DATA Act Information Model Schema (DAIMS)</a>.
\ No newline at end of file
+The DATA Act Broker validates the data it receives from agency financial systems
+ (Files A, B, and C) as well as the Treasury Financial Assistance Broker 
+ Submission (FABS) system. These validations are based on business rules 
+ (such as for award linkage between File C and Files D1 and D2) and checks 
+ against authoritative sources such as GTAS. Agency financial systems are subject 
+ to other data quality assurance measures, such as review by Offices of Inspector
+  General (OIG) and the Government Accountability Office (GAO) for reports as 
+  required by the DATA Act.
+
+USAspending.gov relies on internal validations of data within source systems 
+outside of the Treasury Department (such as FPDS, FSRS, and  <a target="_blank" rel="noopener noreferrer" href="http://sam.gov/" alt="sam.gov link">SAM.gov</a> | Home). 
```

Comment:
Oh, "| Home" should be removed completely.  It's just an artifact from how JIRA interpreted the link.

Reply:
My bad. Done! Changing to a 13point ticket. /s